hub/htype.py

```diff
@@ -67,6 +67,7 @@
     },
     "list": {"dtype": "List"},
     "text": {"dtype": "str"},
+    "dicom": {"sample_compression": "dcm"},
```

Comment:
what if the user passes sample_compression along with the htype?

Reply:
`UnsupportedCompressionError`